client/app/js/crypto/module.js

```diff
@@ -83,28 +87,27 @@ angular.module('GLBrowserCrypto', [])
 // public keys. The attached-model attribute will be bound to the input of the
 // text area.
 .directive('pgpPubkeyInput', function() {
-
   return {
-    restrict: 'E',
+    restrict: 'A',
     templateUrl: '/views/partials/pgp/pubkey_input.html',
     scope: {
       localModel: '=attachedModel',
       keyForm: '=attachedForm',
+      required: '&attachedRequired',
     },
     controller: ['$scope', function($scope) {
-      
-      // The key point of the controller occurs when the keyForm._pubKeyInp input 
+      // The key point of the controller occurs when the keyForm.pgpkey input
       // becomes valid. When that happens the public key attached via localModel
       // is now both ready for use elsewhere in the application and for display
-      $scope.$watch('keyForm._pubKeyInp.$valid', function(newVal, oldVal) {
+      $scope.$watch('keyForm.pgpkey.$valid', function(newVal, oldVal) {
```

Comment:
i know that you liked to keep this private but i think that anyhow we have to make possible for every for for investigating this inside.